src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -180,25 +117,19 @@ export function HistoryEventsTable() {
                 </tr>
               ) : (
                 <>
-                  {eventsHistory.length === 0 &&
-                    eventsHistoryVesting.length === 0 &&
-                    eventsHistoryVestingTeam.length === 0 &&
-                    eventsHistoryVestingOrigin.length === 0 && (
-                      <tr>
-                        <td
-                          colSpan={5}
-                          className="tw-text-center tw-font-normal"
+                  {eventsHistory.length === 0 && (
+                    <tr>
+                      <td colSpan={5} className="tw-text-center tw-font-normal">
+                        <button
+                          type="button"
+                          className="tw-text-gold tw-tracking-normal hover:tw-text-gold hover:tw-no-underline hover:tw-bg-gold hover:tw-bg-opacity-30 tw-mr-1 xl:tw-mr-7 tw-px-4 tw-py-2 tw-bordered tw-transition tw-duration-500 tw-ease-in-out tw-rounded-full tw-border tw-border-gold tw-text-sm tw-font-light tw-font-montserrat"
+                          onClick={() => setViewHistory(true)}
```

Comment:
Create a function with `useCallback` hook and just use the function here.

Reply:
fixed